File: 01_Language/01_Functions/php/gmstrftime.php
<?php

//setlocale(LC_TIME, 'en_US');
ini_set("date.timezone", "Asia/Shanghai");

echo strftime("%b %d %Y %H:%M:%S", mktime(20, 0, 0, 12, 31, 98)) . "\n";
echo gmstrftime("%b %d %Y %H:%M:%S", mktime(20, 0, 0, 12, 31, 98)) . "\n";

File: 01_Language/01_Functions/python/gmstrftime.py
# coding: utf-8

import time


def gmstrftime(fmt, timestamp):
    return time.strftime(fmt, time.gmtime(timestamp))


if __name__ == '__main__':
    print(gmstrftime("%Y-%m-%d %H:%M:%S", time.mktime((1998, 12, 31, 20, 0, 0, -1, -1, -1))))
